samples/glean/metrics.yaml

```diff
@@ -1,22 +1,17 @@
-# This file contains a definition of the scalar probes that are recorded in Telemetry.
-# They are submitted with the "main" pings and can be inspected in about:telemetry.
+$schema: http://mozilla.org/schemas/glean/metrics
 
-# The following section contains the accessibility scalars.
-a11y:
-  instantiators:
-    bug_numbers:
-      - 1323069
-      - 1462238
+browser_engagement:
+  click:
+    type: event
     description: >
-      The leaf name and version number of the binary for the process responsible
-      for remotely instantiating a11y.
-    expires: never
-    kind: string
+      Just testing events
+    bugs:
+      - 123456789
     notification_emails:
-      - accessibility@mozilla.com
-      - jteh@mozilla.com
-    record_in_processes:
-      - 'main'
-    release_channel_collection: opt-out
-    keyed: false
-    cpp_guard: 'ACCESSIBILITY'
\ No newline at end of file
+      - telemetry-client-dev@mozilla.com
+    objects:
```

Comment:
Can we update the sample application as well to make use of these properties?

Reply:
Done.